utilities.py

```diff
@@ -82,3 +87,26 @@ def read_questions(path, id, answerid):
     questions = questions.set_index(id, drop=False)
     questions.sort_index(inplace=True)
     return questions
+
+def _get_auth(env_path):
```

Comment:
@fboylu Can you change the signature from private (_get_auth) to public (get_auth)
In Python functions that start with underscore are considered private and should only be used inside the Python module.
Sorry I should have mentioned that I was using it locally inside my Python module

Reply:
@msalvaris this is fixed now. Thanks.